src/main/resources/lang/fr_FR.yml

```diff
@@ -55,6 +57,26 @@ target:
   msg: Valet cible verrouillé.
 notarget:
   msg: Valet cible déverrouillé.
+help:
+  msg: "1. Tenez l'outil d'édition (<x>) dans votre main principale
+  
+      2. Cliquez à gauche ou à droite à partir d'un support d'armure pour ouvrir le menu.
+      
+      3. Sélectionnez les options de menu marquées.
+      
+      4. Cliquez avec le bouton droit de la souris sur l'armure avec l'outil d'édition en main pour appliquer ces options.
+      
+      5. Cliquez avec le bouton droit de la souris pour modifier à l'envers!"
+helptips:
+  msg: "Astuces:
+  
+  1. Appuyez sur le bouton \"Changer d'Item dans la Main\" (par défaut F) tout en tenant l'outil d'édition pour cibler un support d'armure spécifique, si d'autres étages d'armure sont dans le chemin.
```

Comment:
You should use french quotation marks. As example (you can copy paste if the character is missing from your keyboard layout):

```diff
- 1. Appuyez sur le bouton \"Changer d'Item dans la Main\" (par défaut F) tout en tenant l'outil d'édition pour cibler un support d'armure spécifique, si d'autres étages d'armure sont dans le chemin.
+ 1. Appuyez sur le bouton « changer un objet de main » (par défaut F) tout en tenant l'outil d'édition pour cibler un support d'armure spécifique, si d'autres armure sont sur le chemin.
```
In the meantime, I updated the key name to the one Minecraft uses in its French translation and improved the whole sentence a little bit.